src/ff8/vram.cpp

```diff
@@ -455,7 +521,11 @@ uint32_t ff8_field_read_map_data(char *filename, uint8_t *map_data)
 	snprintf(tex_filename, MAX_PATH, "field/mapdata/%s/%s", get_current_field_name(), get_current_field_name());
 
 	if (save_textures) {
-		ff8_background_save_textures(tiles, mim_texture_buffer, tex_filename);
+		if (save_textures_legacy) {
+			ff8_background_save_textures_legacy(tiles, mim_texture_buffer, tex_filename);
+		} else {
+			ff8_background_save_textures(tiles, mim_texture_buffer, tex_filename);
+		}
```

Comment:
You can probably simplify this condition to something like:
```cpp
if (save_textures_legacy)
  ff8_background_save_textures_legacy(tiles, mim_texture_buffer, tex_filename);
else if (save_textures)
  ff8_background_save_textures(tiles, mim_texture_buffer, tex_filename);
```
This way if someone enables the legacy part only it will dump those, if it enables `save_textures` it will dump the FFNx ones. What do you think?

Reply:
Maybe it makes sense with more legacy texture names, okay fine I make the change